example/php/composer.json

```diff
@@ -20,12 +20,12 @@
   "license": "MIT",
   "type": "library",
   "require": {
-    "php": ">=5.3.0",
-    "monolog/monolog": "1.*"
+    "php": "^7.2"
```

Comment:
thanks for updating this and all the other php files! (I have very little to no experience with it so this is super helpful).